pkg/cmd/cli/backup/delete.go

```diff
@@ -37,6 +39,16 @@ func NewDeleteCommand(f client.Factory, use string) *cobra.Command {
 				os.Exit(1)
 			}
 
+			kubeClient, err := f.KubeClient()
+			cmd.CheckError(err)
+
+			serverVersion, err := kubeutil.ServerVersion(kubeClient.Discovery())
+			cmd.CheckError(err)
+
+			if !serverVersion.AtLeast(controller.MinVersionForDelete) {
+				cmd.CheckError(fmt.Errorf("this command requires the Kubernetes server version to be at least %s", controller.MinVersionForDelete))
```

Comment:
Do you want to do `errors.Errorf` for consistency with the rest of the code base? (Apparently we're still using it in pkg/restore/restore.go when adding warnings & errors to the restore result.)